software/postiz.yml

```
@@ -0,0 +1,10 @@
+name: Postiz
+website_url: https://postiz.com
+description: Schedule posts, track the performance of your content, and manage all your social media accounts in one place (Alternative to Buffer, Hootsuite, Sprout Social).
+licenses:
+  - Apache-2.0
+platforms:
+  - Docker
+tags:
+  - Analytics
+source_code_url: https://github.com/gitroomhq/postiz-app
```

```suggestion
source_code_url: https://github.com/gitroomhq/postiz-app
depends_3rdparty: true
```

As far as I can see all providers under https://docs.postiz.com/providers are non-Free network services so it needs to be specified.